doc/classes/Label.xml

```diff
@@ -14,7 +14,7 @@
 			<return type="Rect2" />
 			<param index="0" name="pos" type="int" />
 			<description>
-				Returns the bounding rectangle of the character at position [param pos]. If the character is a non-visual character or [param pos] is outside the valid range, an empty [Rect2] is returned. If the character is a part of a composite grapheme, the bounding rectangle of the whole grapheme is returned.
+				Returns the bounding rectangle of the character at position [param pos]. The bounding rectangle is interpreted in local 2D coordinates. If the character is a non-visual character or [param pos] is outside the valid range, an empty [Rect2] is returned. If the character is a part of a composite grapheme, the bounding rectangle of the whole grapheme is returned.
```

Comment:
Personally I dislike how _"interpreted"_ is being used here. We know what exactly it returns, so AFAICT there's nothing to be interpreted here.

It's kinda the same wording as for the `Line2D.points` though (mentioned by me in https://github.com/godotengine/godot/issues/95088#issuecomment-2266673908). But there I consider _"interpreted"_ acceptable as `Line2D.points` is a property with a setter, so it makes sense to say how the input (assigned array) will be _"interpreted"_. In case of output/return value I think _"interpreted"_ does not fit (not sure if it's just me though :upside_down_face:).

So I'd rather opt for something like:
```suggestion
				Returns the local bounding rectangle of the character at position [param pos]. If the character is a non-visual character or [param pos] is outside the valid range, an empty [Rect2] is returned. If the character is a part of a composite grapheme, the bounding rectangle of the whole grapheme is returned.
```
or (better):
```suggestion
				Returns the bounding rectangle of the character at position [param pos], in the Label's local coordinate system. If the character is a non-visual character or [param pos] is outside the valid range, an empty [Rect2] is returned. If the character is a part of a composite grapheme, the bounding rectangle of the whole grapheme is returned.
```

AFAICT it's better to be explicit by saying e.g. _"Label's local coordinate system"_ instead of just _"local coordinate system"_ etc. I think doing so should, in the long run, spread the intuition that whatever _"local"_ is always local to some specific object. Which, from my observations, is often not so obvious among the users having problems with differentiating between coordinate systems.

But maybe it would be currently inconsistent with the rest of the docs, not sure. So cc @Mickeon for how exactly we want to word that.

Reply:
I'd say that "interpreted" is appropriate here as it's not so much about ambiguity as it is about what way to treat it, but regardless the wording is clunky to me as well and I'd say "in the ... local coordinate system" is a good wording and AFAIK one that we use elsewhere

Reply:
@kleonc thanks for suggestions! I would go with "Returns the bounding rectangle of the character at position [param pos] in the Label's local coordinate system." (I removed the comma).

@AThousandShips I went crazy and did quick grepping of `returns.*local` and looks like apart "in the ... local coordinate system" sometimes its "using the local coordinate system"
https://github.com/godotengine/godot/blob/74de05a01c8716a42d4e3427f607d7bea76b35e5/doc/classes/CanvasItem.xml#L467

or "in the ... local space"
https://github.com/godotengine/godot/blob/74de05a01c8716a42d4e3427f607d7bea76b35e5/doc/classes/PhysicsServer3D.xml#L942

or "the local ... position "
https://github.com/godotengine/godot/blob/74de05a01c8716a42d4e3427f607d7bea76b35e5/doc/classes/TextEdit.xml#L445